lib/response.js

```diff
@@ -107,24 +107,24 @@ function responseHackMD (res, note) {
     ldapProviderName: config.ldap.providerName,
     saml: config.isSAMLEnable,
     email: config.isEmailEnable,
-    allowemailregister: config.allowemailregister,
-    allowpdfexport: config.allowpdfexport
+    allowemailregister: config.allowEmailRegister,
+    allowpdfexport: config.allowPDFExport
```

Comment:
same as above: the keys are all lowercase, not sure if camelCasing would break anything.

Reply:
For now I stayed on the server side :D But sure we can extend this to the client side as well ^^